### PR TITLE
Add AttendAndSponsorBoxes

### PR DESCRIPTION
**Pull request by warren**
# Description

This PR adds styling (grid-styled and css) to attend and sponsor boxes so they are aligned correctly. It also implements input validation for when a user tries to add themselves to our mailing list.

Also, behind the scenes, it makes sure that every instance of a header is changed to the Header component. We should use this component going forward to maintain style consistency.

# Reviewing

- Try adding an invalid email to the mailing list; it shouldn't let you.
- Try adding a valid email to the mailing list; it should let you.
- After loading the page make sure that attend and sponsor boxes are aligned in the middle of the page, as per the design doc.
- CSS should be finished-level quality and when you press buttons they should have a press animation.

# Tests
I affirm that I've tested my code on:
- [X] Desktop
- [X] Tablet
- [X] Mobile
(Chrome tools counts)

bhacks wip homepage

### CODE REVIEW

File: src/components/EventSchedule/EventSchedule.js [hunk 1]
```diff
@@ -21,20 +22,20 @@ const eventsSun = [
...
 
-const EventScheduleTitle = styled.h2`
-  color: #ef833f;
-  font-family: "Roboto", sans-serif;
-  font-size: 48px;
-  font-weight: 500;
-  position: relative;
-  margin-left: 15%;
-`;
+// const EventScheduleTitle = styled.h2`
```
Comment (lkwatson) [inline src/components/EventSchedule/EventSchedule.js]: If this code is unused, delete

**Review by lkwatson — CHANGES_REQUESTED** [review]
I understand mobile is being implemented after this, but margins are not responsive to middle and large screen sizes. (Looks good on a 13" screen, not so much on an 11" or 15"). See incoming screencaps.

**Comment by lkwatson** [comment]
## Medium screen, no margin between elements. 
screen shot 2018-07-27 at 11 01 57 pm

## Large screen, elements not visible
screen shot 2018-07-27 at 11 02 11 pm

File: src/components/EventSchedule/EventSchedule.js (hunk 1 above)
Comment (maridematte) [reply src/components/EventSchedule/EventSchedule.js]: For the unused code, I will go through the code after most parts are done and clean it up nicely. Some left overs are from works that were supposed to be continued later.

**Comment by warren** [comment]
@lkwatson I'll fix the margin on the medium screen right now but I'm not sure what the correct fix is for the large screen. I understand why it looks wrong but I'm not sure what it would look like if it were right. Could you advise?

If we move the text higher then it's going to clip against the navy swirl.

**Review by maridematte — APPROVED** [review]
It looks good to me, it works nicely on desktop there is only a need to adjust it to mobile.

**Comment by maridematte** [comment]
Also, the margin on top is a little bit too much, and it is also missing the margin on the bottom of the section.

**Comment by lkwatson** [comment]
Re: the text higher, means the curve needs to be more responsive. Merging for now, will need cleanup when all elements are there.

**Review by lkwatson — APPROVED** [review]

**Pull request merged**